scripts/genebuild/create_annotation_configs.pl

```diff
@@ -454,6 +470,20 @@ sub parse_assembly_report {
   $assembly_hash->{'output_path'} .= "/".$species_name."/".$accession."/";
 }
 
+sub update_annotation_status{
+  my $accession = $_[0];
+  my $dt   = DateTime->now;   # Stores current date and time as datetime object
+  my $date = $dt->ymd;
+  my $sql = "insert into genebuild_status(assembly_accession,progress_status,date_started,genebuilder) values(?,?,?,?)";
```

Comment:
MySQL has a function `NOW()` which is better used in this case as we are inserting data in the MySQL database. The current code doesn't need to be changed, this is more of an informative comment

Reply:
Noted